recipe/meta.yaml

```diff
@@ -54,10 +58,12 @@ test:
     - conftest.py
     - setup.cfg
   commands:
-    - pytest --rootdir ibis -m "not (backend or bigquery or clickhouse or hdfs or impala or kudu or mapd or mysql or postgresql or superuser or udf or postgis)" --ignore ibis/tests/test_version.py --tb=line
+    - pytest --rootdir ibis -m "not (backend or bigquery or clickhouse or hdfs or impala or kudu or mapd or mysql or postgresql or superuser or udf or postgis or spark)" --ignore ibis/tests/test_version.py --tb=line
```

Comment:
You shouldn't need to do this with the commit in the recipe, since the spark backend doesn't exist at that commit.